bddl/activity_definitions/cleaning_around_pool_in_garden/problem0.bddl

```diff
@@ -13,7 +13,7 @@
     (:init 
         (covered driveway.n.01_1 dust.n.01_1) 
         (ontop broom.n.01_1 floor.n.01_1)
-        (ontop pool.n.01_1 driveway.n.01_1) 
+        (inroom pool.n.01_1 garden)
```

Comment:
The goal conditions no longer make sense given the task description

Reply:
hmmmm I see your point. Is there any clear way to fix this? l suppose there's no way to explicitly capture the "near pool" semantics in bddl-language, right?

Reply:
conclusion: pool being in same room is sufficient